src/java/org/apache/cassandra/service/MigrationManager.java

```diff
@@ -62,10 +62,13 @@ public class MigrationManager
 
     private static final RuntimeMXBean runtimeMXBean = ManagementFactory.getRuntimeMXBean();
 
+    public static final Set<UUID> scheduledSchemaPulls = new ConcurrentSkipListSet<>();
```

Comment:
Don't love the visibility of this, but we needed access from the migration task. Will probably update the task itself to hold a reference to this.

Reply:
can we just have a function in MigrationManager that MigrationTask calls?